service/controller/v10/templates/cloudconfig/format_etcd_volume.go

```diff
@@ -2,18 +2,21 @@ package cloudconfig
 
 const FormatEtcdVolume = `
 [Unit]
-Description=Formats EBS /dev/xvdh volume
-Requires=dev-xvdh.device
-After=dev-xvdh.device
+Description=Formats EBS volume for etcd
+Before=docker.service var-lib-etcd.mount
 
 [Service]
 Type=oneshot
 RemainAfterExit=yes
 
 # Do not wipe the disk if it's already being used, so the etcd data is
 # persistent across reboots and updates.
-Environment="DEV=/dev/xvdh"
-ExecStart=-/bin/bash -c "if ! blkid $DEV; then wipefs -a -f $DEV && mkfs.ext4 -L etcd $DEV && echo formatted file system; else echo file system already formatted; fi"
+Environment=DEV=/dev/nvme2n1
+
+ExecStart=/bin/bash -c "\
```

Comment:
line 1: for compatibility with `m3.large` that has xvdX disks.
line 2: created filesystem if does not exist
line 3: for compatibility with older clusters will label existing filesystem with `etcd` label

Reply:
Maybe put that in the comment.